docker_images/diffusers/app/pipelines/text_to_image.py

```diff
@@ -18,6 +19,14 @@
 class TextToImagePipeline(Pipeline):
     def __init__(self, model_id: str):
 
+        model_data = model_info(model_id, token=os.getenv("HF_API_TOKEN"))
+        is_lora = any(file.rfilename == 'pytorch_lora_weights.bin' for file in model_data.siblings)
```

Comment:
No `safetensors`  version of it ?  :( :D

Reply:
Great point, I think our training scripts don't yet save `safetensors` lora weights automatically. But I think could be a nice addition! cc @patrickvonplaten @patil-suraj

Reply:
Good point @apolinario , we could support saving `safetensors` in training scripts.